software/spmd/specint2000/README.md

```diff
@@ -9,3 +9,11 @@ SPECInt2000 Benchamarks on Manycore
   contains manycore versions of required raw library functions.
 - `Makefile.BenChMaRK`: Makefile to run `BenChMaRK`
 - `Makefile`: Makefile to run all specint benchmarks
+
+Status:
```

Comment:
if not all the benchmark compiles or run correctly yet, what's the hurry to get this into the master?
Are you simply looking for feedback on the makefile infrastructure you are building?

Reply:
No, I want this on master. It's getting hard to work on branches long term. Especially, when having to run these programs on bsg_bladerunner. 

True, some benchmarks are not compiling because of missing sources and some are failing. But from this repo's point of view, the flow is complete.